### PR TITLE
What techniques are currently used to involve a community with software development? 

### PR DESCRIPTION
**Pull request by WouterKvG**
Not done.. For information only

### CODE REVIEW

File: existing-communities/README.md [hunk 1]
```diff
@@ -1 +1,64 @@
 # Existing Communities
+
+## What techniques are currently used to involve a community with software development?
+There are many projects that are build with the help of an online community. The largest projects include the development of the Linux kernel[^linuxrepo], the Apache project[^apacherepo] and the Mozilla browser[^mozillarepo]. To include the online communities during the development of such online software it is necessary to keep everyone up to date and informed about the current state of the project. In short, the means needed can be divided into five categories: communication, version control, issue tracking, testing and package management `citation? wiki: https://en.wikipedia.org/wiki/Open-source_software_development`. For each of these categories will be invested what techniques are currently used by software development projects.
```
Comment (steffansluis) [inline existing-communities/README.md]: are buil**t** with

Comment (steffansluis) [inline existing-communities/README.md]: `citation? wiki: https://en.wikipedia.org/wiki/Open-source_software_development`

This doesn't seem to be working?

Comment (steffansluis) [inline existing-communities/README.md]: `For each of these categories will be invested what techniques are currently used by software development projects.`

Invested? Do you mean investigated? Also, starting the sentence with 'for' is messing with the structure, I would rephrase it to something like: `The techniques that are currently being used are specified below for each category.`

File: existing-communities/README.md [hunk 2]
```diff
@@ -1 +1,64 @@
...
+
+### Communication
+Communication is perhaps the most important factor when it comes to collaborative software development. Much communication is traditionally done via e-mail, instant messaging, phone, web meetings and groupware {{"thissen2007communication" | cite}}. However as of late new, more enhanced tools like Slack[^slackwebsite] have emerged. With for example Slack it is possible to integrate updates from other development tools such as GitHub and make the visible to the whole team.
```
Comment (steffansluis) [inline existing-communities/README.md]: `With for example Slack it is possible to integrate updates from other development tools such as GitHub and make the visible to the whole team.`

The start of this sentence is a bit weird, I would rephrase it.

File: existing-communities/README.md [hunk 3]
```diff
@@ -1 +1,64 @@
...
+
+### Version control
```
Comment (steffansluis) [inline existing-communities/README.md:9]: Versioning and version control aren't the same thing (are they?)

File: existing-communities/README.md [hunk 4]
```diff
@@ -1 +1,64 @@
...
+Many software project makes use of software versioning. According to the annual Eclipse Community service this percentage totals to almost eighty percent (Git 43.9%, Subversion 30.7%, Other 5.8%) in 2014 {{"eclipse2014survey" | cite}}. From the statistics in the figure below can be concluded that the Git has become the most popular version control method. The main reasons for the growing popularity of Git is its feature set and the usage in large projects such as the aformentioned Linux project {{"stephen2015git" | cite}}.
```
Comment (steffansluis) [inline existing-communities/README.md]: below **it** can be concluded

Comment (steffansluis) [inline existing-communities/README.md]: Git **are** its feature set

File: existing-communities/README.md [hunk 5]
```diff
@@ -1 +1,64 @@
...
+        data2: y2
+axis:
+    y2:
+        show: true
+title:
+    text: "Version control popularity by type"
+{% endchart %}
+
+_Onderstaand stukje kan ook haast als intro gebruikt worden_
+In "Practices in Commercial Projects Using GitHub" by Kalliamvakou et al. the reasons to use GitHub, a populer online Git hosting service, where investigated. Their results yield interesting insights into techniques that can be utilized to enhance the Feedbackfruits platform {{"Kalliamvakou:2015:OSC" | cite}}. The most important findings by Kalliamvakou are:
```
Comment (steffansluis) [inline existing-communities/README.md]: use GitHub, a popul**a**r online

Comment (steffansluis) [inline existing-communities/README.md]: service, **were** investigated

File: existing-communities/README.md [hunk 6]
```diff
@@ -1 +1,64 @@
...
+
+- Reduced communication and coordination needs
+> "Commercial projects use GitHub’s transparency as OSS projects do; their communication is code-centric and pull requests act as a coordination mechanism."
+
+- A touch of self-organization 
+> "Commercial projects use self-organization when assigning tasks and resolving conflicts."
+
+- Challenges in collaborating with non-technical members
+> "Commercial projects resort to using external tools when collaborating with non-technical members operating outside GitHub."
+
```
Comment (steffansluis) [inline existing-communities/README.md]: These findings aren't really worded like they are findings in my opinion. They sound more like principles or the start of a sentence

**Comment by WouterKvG** [comment]
@steffansluis @bartist Please review. I am quite happy with the current text. Maybe the 'Other thoughts' section can be improved. Do you have any suggestions? Furthermore this section should probably be moved to another separate file.

File: existing-communities/README.md [hunk 7]
```diff
@@ -1 +1,47 @@
...
+        data2: y2
+axis:
+    y2:
+        show: true
+title:
+    text: "Version control popularity by type"
+{% endchart %}
+
+### Issue tracking
+The quote "Software does not allow itself to be finished. Software is never finished" by Paul LeBlanc {{"leblanc1993writing" | cite}} shows the need for good issue tracking. There is an enormous list of issue-tracking solutions available {{"multiple2016issuetracking" | cite}}, each with their own strengths and weaknesses. Interviews conducted by Kalliamvakou show that for example the combination of the GitHub issue-tracker enhances self-organization when assiging tasks and resolving conflicts {{"Kalliamvakou:2015:OSC" | cite}}. However the GitHub solution is not optimal because the issue tracker lacks some critcial features such as severity, component and a bug report template {{"Zhou:2015:CAB:2745802.2745808" |  cite}}.
```
Comment (steffansluis) [inline existing-communities/README.md]: `as severity, component and a bug report template`

This should all be pluralized I think, so `as severity, components and a bug report templates`
